src/components/tx/TxSimulation/utils.ts

```diff
@@ -99,14 +100,20 @@ export const _getSingleTransactionPayload = (
     transaction = simulatedTransaction
   }
 
+  // If gasLimit <= safeTxGas we reduce safeTxGas 10% in the simulation payload
+  let adjustedSafeTxGas
+  if (gasLimit <= transaction.data.safeTxGas) {
+    adjustedSafeTxGas = transaction.data.safeTxGas * 0.9
+  }
+
   const readOnlySafeContract = getReadOnlyCurrentGnosisSafeContract(params.safe)
 
   const input = readOnlySafeContract.encode('execTransaction', [
     transaction.data.to,
     transaction.data.value,
     transaction.data.data,
     transaction.data.operation,
-    transaction.data.safeTxGas,
+    adjustedSafeTxGas || transaction.data.safeTxGas,
```

Comment:
Shouldn't we also use this value for the transaction? The simulation otherwise differs to what is executed.